chef/cookbooks/ceph/recipes/mds.rb

```diff
@@ -0,0 +1,47 @@
+if node.roles.include?("ceph-mon")
+  include_recipe "ceph::default"
+  include_recipe "ceph::conf"
+else
+  include_recipe "ceph::keyring"
+end
+include_recipe "ceph::server"
+
+execute "create mds keyring" do
+  command "ceph auth get-or-create mds.#{node["hostname"]} osd 'allow rwx' mds 'allow' mon 'allow profile mds' -o /etc/ceph/mds.#{node["hostname"]}.keyring ; chown ceph.ceph /etc/ceph/mds.#{node["hostname"]}.keyring"
```

Comment:
So here is what I think to understand: As mentioned in the issue the `*` permission is needed to issue `mds tell` commands to an MDS. `tell` is used to change config parameters of an MDS while it is running...so the permissions should be ok. I assume an MDS won't change its own config or that of other MDS's.